galaxy/ansible/templates/export_user_files.py.j2

```diff
@@ -142,7 +142,7 @@ if __name__ == "__main__":
     change_path( os.path.join('/tool_deps') )
     change_path( os.path.join(galaxy_root_dir, 'tool-data') )
     change_path( os.path.join(galaxy_root_dir, 'database') )
-    change_path( '/shed_tools/' )
+    change_path( '/tus_upload_store/' )
 
```

Comment:
Tus upload store are temporary files during the upload. I don't think we need to persist them.

Reply:
will remove it